tools/tf2openapi/types/tftensor.go

```diff
@@ -34,30 +36,79 @@ type TFDType int
 const (
 	// all the possible constants that can be JSON-ified according to
 	// https://www.tensorflow.org/tfx/serving/api_rest#json_mapping
-	DT_BOOL TFDType = iota
-	DT_B64_STRING
-	DT_STRING
-	DT_INT8
-	DT_UINT8
-	DT_INT16
-	DT_INT32
-	DT_UINT32
-	DT_INT64
-	DT_UINT64
-	DT_FLOAT
-	DT_DOUBLE
+	// along with a representation for B64 strings
+	DtBool TFDType = iota
+	DtB64String
+	DtString
```

Comment:
I tried to adhere to the Golang naming conventions - The standard library uses camel-case for constants instead of upper-case (https://stackoverflow.com/a/22688926) and the effective Go guide (https://golang.org/doc/effective_go.html#mixed-caps) says to use mixed caps rather than underscores

Reply:
Got it. Thanks for the pointers!